src/dapps/aave-dapp/components/AaveDepositOverlay.vue

```diff
@@ -95,8 +95,7 @@ export default {
           return item;
         }
       });
-
-      return hasBalance ? BigNumber(hasBalance.totalBalance).toFixed() : '0';
+      return hasBalance ? BigNumber(hasBalance.usdBalance).toFixed() : '0';
```

Comment:
we need to  change this to the formatFiatValue, but we can probably do it once we work on aave separetly